### DISCUSSION TITLE
Different event colours not showing

### DISCUSSION THREAD

**Discussion by peggydawg** [kewisch/gdata-provider#455] [Support]
Hi there, 

In a google calendar that was shared with me, and in which I have edit permissions, events that show up as different colours in google calendar only show up as one on Thunderbird; any ideas as to how I can solve this? It's pedantic, arguably, but important in the context of the calendar.

Thanks!

Provider 102.0.0
Thunderbird 102.0b8
Windows 10

**Comment by derenv** [comment]
Yeah i would love support for this, colours for different events are great visual indicators.

**Comment by kewisch** [comment]
Thunderbird doesn't support different colors within a calendar, so there is unfortunately nothing I can do. You could file an enhancement to Thunderbird to make this possible, when that is fixed I can make use of it. Sorry I can't give you better news!
